README.md

```diff
@@ -74,17 +99,13 @@ To install Mapbox Navigation using [Carthage](https://github.com/Carthage/Cartha
    ```cartfile
    # Latest stable release
    github "mapbox/mapbox-navigation-ios" ~> 1.2
-   # Latest prerelease
-   github "mapbox/mapbox-navigation-ios" "v1.2.0-rc.1"
    ```
 
 1. Run `./Carthage/Checkouts/mapbox-navigation-ios/scripts/wcarthage.sh bootstrap --platform iOS --cache-builds --use-netrc`. (wcarthage.sh is a temporary replacement for `carthage` to work around [a linker error in Xcode 12](https://github.com/Carthage/Carthage/issues/3019).)
 
 1. Follow the rest of [Carthage’s iOS integration instructions](https://github.com/Carthage/Carthage#if-youre-building-for-ios-tvos-or-watchos). Your application target’s Embed Frameworks build phase should include `MapboxNavigation.framework`, `MapboxCoreNavigation.framework`, `MapboxNavigationNative.framework`, `MapboxCommon.framework`, and `MapboxAccounts.framework`.
 
-### Using Swift Package Manager
-
-The MapboxNavigation framework [does not yet support Swift Package Manager](https://github.com/mapbox/mapbox-navigation-ios/issues/2629) either on the command line or in Xcode. However, if you are building a user interface from scratch, you can [install just the MapboxCoreNavigation framework using SPM](./custom-navigation.md#using-swift-package-manager). 
+The MapboxNavigation framework no longer supports Carthage. However, if you are building a user interface from scratch, you can [install just the MapboxCoreNavigation framework using Carthage](./custom-navigation.md#using-carthage).
```

Comment:
If this is true, why is the section `### Using Carthage` still present?

Reply:
There are two separate frameworks, MapboxNavigation does not support Carthage but MapboxCoreNavigation continues to support it.

Reply:
Im still a bit confused, if MapboxNavigation does not support carthage why does it say:

> ### Using Carthage
> To install the MapboxNavigation framework using [Carthage](https://github.com/Carthage/Carthage/) v0.35 or above [...]

Reply:
The readme you’re looking at has been updated a couple times since this PR landed. [Here’s the current version](https://github.com/mapbox/mapbox-navigation-ios/tree/release-v2.0#using-carthage) from the release-v2.0 branch, where we’re developing v2.0.0. [The version on the main branch](https://github.com/mapbox/mapbox-navigation-ios/tree/main#using-carthage) does still say that MapboxNavigation supports Carthage, because that’s true of v1._x_.

Reply:
Thanks, got it! :) Looks like the new version has a helpful qualifier
> MapboxNavigation no longer supports Carthage as of v2.0.0-alpha.1